frontend/src/app/pages/creator/creator.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { UserDTO } from 'src/app/core/models/user-dto';
+import { VideoMetadataDto } from 'src/app/core/models/video-metadata-dto';
+import { UserService } from 'src/app/core/services/user.service';
+import { VideoService } from 'src/app/core/services/video.service';
+
+@Component({
+  selector: 'app-creator',
+  templateUrl: './creator.component.html',
+  styleUrls: ['./creator.component.scss']
+})
+export class CreatorComponent {
+  videosPerRow: number = 6;
```

Comment:
why is it like that? cant we use grid (I think it is grid) and choose dynamicaly how many videos per row are visible?

Reply:
the whole logic regarding what video should be in what row will be unncecessary